runtime/doc/options.txt

```diff
@@ -3799,9 +3799,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 <							*lcs-leadmultispace*
 	  leadmultispace:c...
 			Like multispace value, but only for leading whitespace
-			Overrides |lcs-lead| for leading multiple spaces.
-			`:set listchars=leadmultispace:---+` shows ten consecutive
-			leading spaces as:
+			overrides |lcs-lead| for leading multiple spaces.
```

Comment:
I think what is actually needed is a  `.` at the end of the first line, like:
```vimhelp
			Like multispace value, but only for leading
			whitespace. Overrides |lcs-lead| for leading multiple
			spaces. `:set listchars=leadmultispace:---+` shows ten
			consecutive leading spaces as:
```
Though it does cause a bit of wrapping.